bundle/Controller/Admin/AdminController.php

```diff
@@ -271,10 +293,87 @@ public function handleCollectionAction(Request $request): RedirectResponse
         return $this->redirectToRoute('netgen_information_collection.route.admin.view', ['collectionId' => $collectionId]);
     }
 
+    /**
+     * @throws NonUniqueResultException
+     */
+    public function editAction(Request $request, int $collectionId): RedirectResponse|Response
+    {
+        $this->checkEditPermissions();
+
+        $collection = $this->service->getCollection(new CollectionId($collectionId));
+
+        $location = $collection->getContent()->contentInfo->getMainLocation();
+
+        $form = $this->builder->createUpdateFormForLocation($location, $collection)->getForm();
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            /** @var InformationCollectionStruct $struct */
+            $struct = $form->getData()->payload;
+
+            /** @var ContentType $contentType */
+            $contentType = $form->getData()->definition;
+
+            $ezInfoCollection = $this->infoCollectionRepository->find($collectionId);
+
+            if ($ezInfoCollection === null) {
+                throw new NotFoundHttpException();
+            }
+
+            $ezInfoCollection->setModified(time());
+
+            $this->infoCollectionRepository->save($ezInfoCollection);
+
+            foreach ($struct->getCollectedFields() as $fieldDefIdentifier => $value) {
+                if ($value === null) {
+                    continue;
+                }
+
+                $fieldDefinition = $contentType->getFieldDefinition($fieldDefIdentifier);
```

Comment:
Make sure that `$fieldDefinition` is not null.

Reply:
fixed in commit 36d5f6a8